packages/extension-chakra-storefront/babel.config.js

```diff
@@ -4,4 +4,4 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-module.exports = require('internal-lib-build/configs/babel.config')
+module.exports = require('@salesforce/pwa-kit-dev/configs/babel/babel-config')
```

Comment:
This was just a mistake. Templates and Applications should be using the `@salesforce/pwa-kit-dev` package for building and config.